### PR TITLE
Plane wiki: add QACRO mode and "copter" tailsitters and bodyframe-roll mode

### PR DESCRIPTION
**Pull request by kd0aij**
still needs additions for the new tailsitter class and control mode

### CODE REVIEW

**Comment by kd0aij** [comment]
@tridge  @IamPete1 Please take a look at the bodyframe roll description and tell me what you think of the roll and yaw parameters being swapped. Do you think additional description of the parameters would be sufficient, or does it require code changes to be acceptable?

File: plane/source/docs/qacro-mode.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+It is recommended to set parameter Q_TAILSIT_INPUT to FixedWing (1) for this mode. This selects plane style controls with the aileron stick controlling body frame roll rate and the rudder stick controlling body frame yaw rate.
+
+Also, set Q_M_HOVER_LEARN to 2 (learn and save) to allow the throttle midpoint to set automatically.
+
+Training mode is not implemented and this mode will behave similarly to a fixed wing plane in MANUAL mode, with no limitations on earth frame roll, pitch and yaw.
+
+Aileron and elevator stick scalings are set by parameters ACRO_ROLL_RATE and ACRO_PITCH_RATE, respectively, with default of 180 deg/sec. Yaw stick scaling is set by Q_YAW_RATE_MAX with default of 100 deg/sec.
+
+To avoid oscillation, it is necessary to reduce control surface deflections at high airspeeds in VTOL modes.  If the vehicle has no airspeed sensor, this reduction is based on attitude (tilt angle from vertical) and throttle setting, since these are generally correlated with airspeed. If an airspeed sensor is available, the reduction is based on the measured airspeed.
```
Comment (IamPete1) [inline plane/source/docs/qacro-mode.rst]: i think this needs a sub title, "For Tailsitters" or something

File: plane/source/docs/qacro-mode.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.. _qacro-mode:
+
+=========+
+QACRO Mode
+=========+
+
+QACRO (for acrobatic) is a quadplane mode for advanced users that provides rate
+based stabilization like :ref:`Copter ACRO <copter:acro-mode>`.  QACRO mode is best suited for aerobatic flight of a copter tailsitter.
+
+It is recommended to set parameter Q_TAILSIT_INPUT to FixedWing (1) for this mode. This selects plane style controls with the aileron stick controlling body frame roll rate and the rudder stick controlling body frame yaw rate.
```
Comment (IamPete1) [inline plane/source/docs/qacro-mode.rst]: this bit should probably be down with the other tailsitter stuff. Does it abide by Q_TAILSIT_INPUT  or just assume its 1?

Comment (kd0aij) [reply plane/source/docs/qacro-mode.rst]: thanks, that's just wrong... input_type does not apply to QACRO at all.  I hope I got the description right on the tailsitter page.
QACRO isn't restricted to tailsitters, so I think this belongs here, though.

File: plane/source/docs/qacro-mode.rst (hunk 1 above)
Comment (kd0aij) [reply plane/source/docs/qacro-mode.rst]: I'm working on a PR to remove the confusion between roll and yaw, and add a new parameters as necessary to get the units right.

**Comment by rmackay9** [comment]
It's a bit out of my jurisdiction to merge this - but I suspect it's better to go in and leave it out..

**Comment by kd0aij** [comment]
Thanks Randy, I'll submit updates as this evolves

**Pull request merged**